src/edwards.rs

```diff
@@ -690,14 +690,14 @@ impl ExtendedPoint {
     /// Returns `Some<[u8;32]>` if `self` is in the image of the
     /// Elligator2 map.  For a random point on the curve, this happens
     /// with probability 1/2.  Otherwise, returns `None`.
-    pub fn to_uniform_representative(&self) -> Option<[u8; 32]> {
+    fn to_uniform_representative(&self) -> Option<[u8; 32]> {
```

Comment:
fwiw, we can probably just remove these entirely since I was planning to refactor my elligator2 code to implement `std::convert::From`.